spec/hockey_gerrit_spec.rb

```diff
@@ -1,5 +1,68 @@
-require_relative 'spec_helper'
+require_relative 'helper/spec_helper'
 
 describe HockeyGerrit do
-  # TODO: write tests
+  let(:gerrit_refspec) { 'refs/changes/13/70000/10' }
+  let(:hockey) { HockeyGerrit.new }
+
+  def hockey_run(opts = {})
+    ipa = opts.fetch :ipa, ipa_path
+    tries = opts.fetch :retry, 1
+    hockey.run(token: '123',
+               ipa: ipa,
+               build_url: '345',
+               gerrit: gerrit_refspec,
+               retry: tries)
+  end
+
+  before do
+    allow(hockey).to receive(:git_log) { 'bootstraponline: Stub git log' }
+    allow(hockey).to receive(:git_commit_sha) { '123456789641603afe0572f3b4a91204791b012d' }
+  end
+
+  it 'posts app to hockey' do
+    stub_valid_response
+
+    hockey_run
+
+    expected_url = "#{RINK_HOCKEY_URL}/manage/apps/123456/app_versions/9"
```

Comment:
/manage/apps/123456/app_versions/9
is used more than once. Can we promote it to a global in the spec_helper too?